flatliners/clusteralertcorr.py

```diff
@@ -82,9 +83,13 @@ def on_next(self, x):
                     alert_combination_list.append('_'.join(elm))
                     alert_combination_value_list.append(val)
 
-                self.clusters_correlation[clust_id] = pd.DataFrame(data = [alert_combination_value_list], columns = alert_combination_list)
+                self.clusters_correlation[clust_id] = {'dataframe': pd.DataFrame(data = [alert_combination_value_list],
+                                                                    columns = alert_combination_list),
+                                                       'version': version_id}
```

Comment:
Is version Id saving required here? If it is, I would suggest for re-using the method from Baseflatliner, "cluster_version".

Reply:
I think it is required @subhojit-mukherjee , as the the next processor ` corrcomparison.py` needs this information to pair clusters and versions.